README.md

```diff
@@ -1,2 +1,70 @@
-# bot-template
-A basic template for a discord bot with disnake.
+# Basic Bot Template
+
+<div style="text-align: center;">
+
+![GitHub Issues or Pull Requests](https://img.shields.io/github/issues/OseSem/bot-template)
+![GitHub License](https://img.shields.io/github/license/OseSem/bot-template)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/w/OseSem/bot-template)
+
+</div>
+
+This is a basic bot template for the [Discord API](https://discord.com/developers/docs/intro) using [disnake](https://docs.disnake.dev).
+
+
+## Key Features
+- Neat packaging control.
+- Workflows for linting.
+- Basic bot setup.
+
+## Getting Started
+1. Clone the template ***[here](https://github.com/new?template_name=bot-template&template_owner=OseSem)***.
+2. Installing the dependencies: (Remove `--without dev` if you want to install dev dependencies)
+```bash
+python3 -m pip install poetry
+poetry install
+```
+3. Create a `.env` file in the root directory and add the following:
+```env
+TOKEN=YOUR_BOT_TOKEN
+```
+4. Update version and title in `pyproject.toml` and `src/__init__.py`.
+5. Run the bot: (Run `exit` to exit the shell.`)
+```bash
+poetry shell
+python3 main.py
+```
+
+## Examples
+### Making a cog:
```

Comment:
Plugins should be called plugins, not cogs

Reply:
Fixed.